atom/browser/ui/certificate_trust_win.cc

```diff
@@ -79,7 +80,8 @@ void ShowCertificateTrust(atom::NativeWindow* parent_window,
     CertFreeCertificateChain(chain_context);
   }
 
-  callback.Run();
+  promise.Resolve();
```

Comment:
Is this method is synchronous / blocking on windows?  Might be worth documenting / possibly making async somehow 🤔